basterdz-api/src/main/java/com/dnd/api/domains/room/service/RoomService.java

```diff
@@ -55,16 +57,26 @@ public Room enterRoom(final String inviteCode, final Member member) {
         room.addMemberCount();
 
         roomMemberAppender.append(roomMember);
-        return roomAppender.append(room);
+        return room;
+    }
+
+    @Transactional
+    public Room startRoom(final Member member, final Long roomId) {
+        Room room = findRoom(roomId);
+        RoomMember roomMember = roomMemberFinder.findRoomMember(member, room);
+        if (roomMember.isHost() == IS_HOST) {
+            room.activate(ACTIVE);
+        }
```

Comment:
```suggestion
        if (roomMember.isHost()) {
            room.activate(ACTIVE);
        }
```
추가적으로 만약 ACTIVE를 파라미터로 넘긴다면 메서드명이 changeStatus가 맞아보입니다.